cloudflare-lb/cloudflare_lb_module/variables.tf

```diff
@@ -0,0 +1,133 @@
+variable "account_id" {
+  type        = string
+  description = "Cloudflare account id"
+  sensitive   = true
+}
+
+variable "zone_id" {
+  type        = string
+  description = "Cloudflare zone id"
+  sensitive   = true
+
+}
+
+variable "cloudflare_api_token" {
+  type        = string
+  description = "Cloudflare API token"
+  sensitive   = true
+}
+
+variable "domain" {
+  type    = string
+  default = "subspace.network"
+}
+
+variable "network" {
+  type        = string
+  description = "gemini network name"
+}
+
+variable "records" {
+  description = "List of DNS records."
+  type = list(object({
+    name     = string
+    hostname = string
+    value    = string
+    type     = string
+    tags     = list(string)
+  }))
+  default = [
+    {
+      name     = "rpc-0"
+      hostname = "rpc.gemini-3f.subspace.network"
```

Comment:
Shouldn't these be coming from the project that uses this module instead ?
Seems weird seeing gemini-3f as default under a terraform module

Reply:
Yeah this has been removed, missed it when refactoring to module.